split.Console/Interfaces/IDeckInterface.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a rough draft. It will be implemented by an object
+    /// that encapsulates the actions of a Card deck. 
+    /// </summary>
+    public interface IDeckInterface
+    {
+        IDeckInterface shuffle();
```

Comment:
Sounds good, just be careful to avoid duplication between different implementations.